samples/drivers/w1/scanner/sample.yaml

```diff
@@ -0,0 +1,25 @@
+sample:
+  name: 1-Wire scanner sample
+
+common:
+    harness: console
+
+tests:
+  samples.drivers.w1.scanner.ds2484:
+    depends_on: arduino_i2c
+    extra_args: DTC_OVERLAY_FILE=ds2484.overlay OVERLAY_CONFIG=ds2484.conf
+    platform_allow: nrf52840dk_nrf52840
+    harness_config:
+      type: one_line
+      regex:
+        - "Number of devices found on bus: .*"
+      fixture: w1_scanner_ds2484
+  samples.drivers.w1.scanner.w1_serial:
+    depends_on: arduino_serial
+    extra_args: DTC_OVERLAY_FILE=w1_serial.overlay OVERLAY_CONFIG=w1_serial.conf
+    platform_allow: nrf52840dk_nrf52840
+    harness_config:
+      type: one_line
+      regex:
+        - "Number of devices found on bus: .*"
```

Comment:
You could move these lines to the common section.

Reply:
If I add `harness_config` to the common section, the harness_config sections under `tests:` are ignored and therefore the fixtures are not respected.